pycm/pycm_output.py

```diff
@@ -353,22 +353,32 @@ def sparse_table_print(sparse_matrix):
     return result
 
 
-def csv_matrix_print(classes, table):
+def csv_matrix_print(classes, table, header=False):
     """
     Return matrix as csv data.
 
     :param classes: classes list
     :type classes:list
     :param table: table
     :type table:dict
+    :param header: add headers to .csv file
+    :type header: bool
     :return:
     """
     result = ""
+    header_section = ""
     classes.sort()
     for i in classes:
+        if header is True:
+            header_section += i
+            if i != classes[-1]:
```

Comment:
Logic is right, but this condition will check in **each iteration**, I think it should be check at the end of this function (only a single if) and out of any loop ;-)

Reply:
Yeah i though about this alternative, i thought this approach won't make header section in the case that we don't need it.
but as you say i'll change it.

Reply:
> Yeah i though about this alternative, i thought this approach won't make header section in the case that we don't need it.
> but as you say i'll change it.

I only mentioned second `if ... else` (adding `,` and `n`), the first `if` position was true and you are right about it ;-)